staging/script.xsqueeze/resources/lib/classes/NowPlayingWindow.py

```diff
@@ -8,6 +8,13 @@
 import datetime
 import constants
 
+#hdmi support
+if constants.CECSUPPORT == 'true':
+  import cec
+  cec.init()
+  tv   = cec.Device(0)
+  avr  = cec.Device(5)
+
```

Comment:
I think this will also work for other. 
The ID "0" is always a TV and "5" is always an Audio System.
ARC can be configured in the settings of the TV.